pkg/arvo/sys/lull.hoon

```diff
@@ -462,7 +462,7 @@
 +$  rift  @ud                                           ::  ship continuity
 +$  mime  (pair mite octs)                              ::  mimetyped data
 +$  octs  (pair @ud @)                                  ::  octet-stream
-+$  sock  (pair ship ship)                              ::  outgoing [our his]
++$  sock  (trel ship ship path)                         ::  [src dest prov]
```

Comment:
I'm inclined to think we'd be better off making a new type for this and leaving `$sock` as-is, to reduce the amount of work client code needs to change.

`$sack` could be the name for the new type, perhaps -- it's not used elsewhere afaik.

Reply:
added $sack as recommended